x-pack/test/security_solution_endpoint/page_objects/policy_page.ts

```diff
@@ -27,7 +27,7 @@ export function EndpointPolicyPageProvider({ getService, getPageObjects }: FtrPr
      */
     async findFirstActionsButton() {
       await this.ensureIsOnPolicyPage();
-      return (await testSubjects.findAll('policyActionsButton'))[0];
+      return await testSubjects.find('policyActionsButton');
```

Comment:
`testSubjects.find` wraps `driver.findElement` that will return the first element in DOM with specified locator and throw exception in case it is not in DOM. It is better than searching for an array and selecting by index when array can be empty.